README.md

```diff
@@ -4,6 +4,10 @@ Simple tool to validate and lint `metadata.json` files in Puppet modules as
 recommended in Puppet Forge style guidelines from [Puppet forge metadata style
 quide](https://docs.puppetlabs.com/puppet/latest/reference/modules_publishing.html#write-a-metadatajson-file)
 
+## Compatibility
+
+metadata-json-lint is compatible with Ruby versions 2.1.x and 2.3.x.
```

Comment:
Is there a reason we would skip 2.2.x? It wouldn't add a lot of test time to add it to the matrix.